### PR TITLE
Add option for helm deployments to create namespace

### PR DESCRIPTION
**Pull request by anshlykov**
Fixes: #4482

**Description**
This PR adds the ability to create a namespace for Helm deploy command if Helm version is 3.2+.

### CODE REVIEW

File: pkg/skaffold/deploy/helm.go [hunk 1]
```diff
@@ -506,6 +507,10 @@ func installArgs(r latest.HelmRelease, builds []build.Artifact, valuesSet map[st
 		args = append(args, "--namespace", o.namespace)
 	}
 
+	if r.CreateNamespace && o.helmVersion.GE(helm32Version) && !o.upgrade {
```
Comment (gsquared94) [inline pkg/skaffold/deploy/helm.go:510]: can we fail fast with an appropriate error message when `CreateNamespace` is specified but the min required helm version is not met? You'd have to make it `*bool` in `config.go`

File: pkg/skaffold/deploy/helm_test.go [hunk 2]
```diff
@@ -272,6 +272,7 @@ var testTwoReleases = latest.HelmDeploy{
 	}},
 }
 
+var crateNamespaceFlag = true
```
Comment (gsquared94) [inline pkg/skaffold/deploy/helm_test.go:275]:
```suggestion
var createNamespaceFlag = true
```

File: pkg/skaffold/deploy/helm_test.go [hunk 3]
```diff
@@ -284,7 +285,7 @@ var testDeployCreateNamespaceConfig = latest.HelmDeploy{
 			"some.key": "somevalue",
 		},
 		Namespace:       "testReleaseNamespace",
-		CreateNamespace: true,
+		CreateNamespace: &crateNamespaceFlag,
```
Comment (gsquared94) [inline pkg/skaffold/deploy/helm_test.go:288]:
```suggestion
		CreateNamespace: &createNamespaceFlag,
```

File: pkg/skaffold/deploy/helm_test.go (hunk 2 above)
Comment (anshlykov) [reply pkg/skaffold/deploy/helm_test.go]: @gsquared94 thanks for catching this. fixed

**Review by gsquared94 — APPROVED** [review]

**Pull request merged**